ampache/Caddyfile

```diff
@@ -0,0 +1,69 @@
+ampache.domain.com {
+    root /home/caddy/web/ampache.domain.com
+
+    log /home/caddy/web/log/ampache.domain.com.access.log {
+        rotate_size 100  # Rotate after 100 MB
+        rotate_age  14   # Keep log files for 14 days
+        rotate_keep 10   # keep maximum of 10 lof files
+    }
+
+    errors /home/caddy/web/log/ampache.domain.com.error.log
+
+    index index.php
+
+    gzip
+
+    fastcgi / unix:/var/run/php/php7.0-fpm.sock php
+
+    # Rewrite rules for Subsonic backend
+    rewrite /rest {
+        r ^/rest/(.*).view$
+        to {path}/ /rest/index.php?action={1}
+    }
+    rewrite /rest/fake {
+        r ^/rest/fake/(.+)$
+        to {path}/ /play/{1}
+    }
+
+    # Rewrite rule for Channels
+    rewrite /channel {
+        r ^/channel/([0-9]+)/(.*)$
+        to /channel/index.php?channel={1&}target={2} last;
+    }
+
+    # Beautiful URL Rewriting
+    rewrite /play/ssid {
```

Comment:
The base path is not needed when using regexes. @abiosoft Maybe that should be an error too?

(This goes for all the rewrite blocks)

Reply:
The base path actually works. And it's an easy way for readability to know which subpath the regex is matching. And when base path are used, only the subpath is matched with regex.

Reply:
Oh! I didn't realize that. Cool. Okay, then they can stay.

Reply:
Maybe the docs need to be more explicit.

Reply:
This is not correct nonetheless. The regex configs need to omit the base path. i.e. `/play/ssid` in this case.

Reply:
Hi guys, thanks for the feedback. I must say I decided not to go forward with Ampache. The software doesn't fit my needs. I still have it running though, so i'll fix your comments and test if it still works. After that i'll update the PR :)

Reply:
Okay. Thanks for contributing it anyway! I'll let @abiosoft handle this PR then, if that's alright. Sounds like we may be working on a minor change in Caddy either way related to it.